src/main/java/cloud/eppo/ConfigurationStore.java

```diff
@@ -20,9 +20,20 @@ public class ConfigurationStore {
   private Map<String, BanditParameters> banditParameters;
 
   public ConfigurationStore() {
+    this(null, null);
+  }
+
+  public ConfigurationStore(String initialFlagConfiguration, String initialBanditParameters) {
```

Comment:
It probably needs to store obfuscation flag as well. Otherwise, it's likely susceptible to a bug: when it is initialized with non-obfuscated config and fetches an obfuscated config later on, for a short duration it might either interpret old configuration as obfuscated, or interpret the new configuration as non-obfuscated, leading to evaluation errors

Reply:
Good catch.
With the move to `ConfigurationBuffer`, I've added a field for `isConfigObfuscated`.
`EppoClient` can be initialized with an obfuscated configuration (or not), independent of whether it should expect obfuscated config from the API server.